steward/src/utils.rs

```diff
@@ -32,3 +38,111 @@ pub async fn get_delegates_keys_by_orchestrator(
 
     Ok(keys)
 }
+
+/// Verify that a url has an http or https prefix
+fn check_scheme(input: &Url, original_string: &str) {
+    if !(input.scheme() == "http" || input.scheme() == "https") {
+        panic!(
+            "Your url {} has an invalid scheme, please chose http or https",
+            original_string
+        )
+    }
+}
+
+pub async fn get_eth_provider(eth_rpc_url: &str) -> Result<Provider<Http>, Error> {
```

Comment:
I don't think we need to copy-paste this sort of mess of a setup from the bridge code. We should simply try or fail given the RPC URL we are provided.

Reply:
I'm glad you said that lol.